### DISCUSSION TITLE
Como fazer isso no React JS?

### DISCUSSION THREAD

**Discussion by andregzanon** [frontendbr/forum#2357] [Perguntas e Respostas]
Como fazer isso no React **JS**

![image](https://user-images.githubusercontent.com/78750480/193415384-e7b590cc-2f3c-417d-85f3-04d846d7141c.png)

**Comment by julianosirtori** [comment]
Vc pode detalhar mais a pergunta?
Nao ficou claro o q vc precisa fazer com o React, fazer um fluxograma? fazer um fluxo com um botao? fazer um hello world?

**Reply by andregzanon** [reply]
Criar um fluxograma, para um aplicação que estou fazendo.. se possível em react JS.

Eu já encontrei https://reactflow.dev/ mas ele é em Typescript

**Reply by ghost** [reply]
Mas se é em Typescript, creio que funcionaria com JavaScript usando sem a tipagem.

**Reply by didiraja** [reply]
@andregzanon 
o exemplo é sem typescript https://reactflow.dev/docs/examples/overview/

**Comment by lucas-nogueira-leggal** [comment]
Estou fazendo o mesmo projetos, podemos compartilhar conhecimento, também estou utilizando essa biblioteca. Conseguiu utilizar ela ?

**Comment by geanrt** [comment]
Utilizem o React Flow, é complicado pra caramba, assim como qualquer drag-and-drop (eu particulamente passo muito tempo nisso, e é que tenho muito experiencia).
